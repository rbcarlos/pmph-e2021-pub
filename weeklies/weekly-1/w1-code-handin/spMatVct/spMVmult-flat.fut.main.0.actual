[0.0f32, 0.0f32, 0.0f32, 0.0f32, 0.0f32]
