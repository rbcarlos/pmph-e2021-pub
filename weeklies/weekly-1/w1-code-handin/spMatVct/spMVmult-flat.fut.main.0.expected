[3.0f32, 0.0f32, -4.0f32, 6.0f32, 9.0f32]
